lms/djangoapps/commerce/api/v1/permissions.py

```diff
@@ -1,12 +1,32 @@
 """ Custom API permissions. """
+from django.conf import settings
+from django.contrib.auth.models import User
 from rest_framework.permissions import BasePermission, DjangoModelPermissions
+import waffle
 
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 from openedx.core.lib.api.permissions import ApiKeyHeaderPermission
 
+DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH = configuration_helpers.get_value(
+    'DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH',
+    settings.DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH
+)
```

Comment:
I just realized this won't work the way we want it to. The `DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH` variable will get set at startup outside the context of the request. You would need to move down closer to where you are using it within the request context.